README.md

```diff
@@ -1,8 +1,8 @@
 # biahub
 
-Bio-image analysis hub supporting high-throughput data reconstruction on HPC clusters with [Slurm](https://slurm.schedmd.com/documentation.html) workload management.
+biahub (bioimage analysis hub) is a pipeline to turn raw high throughput imaging data into AI ready nD datasets on HPC clusters running on [Slurm](https://slurm.schedmd.com/documentation.html) workload management.
 
-![acquisition and reconstruction schematic](https://github.com/czbiohub-sf/shrimPy/blob/main/docs/figure_3a.png)
+![biahub schematic](https://github.com/czbiohub-sf/shrimPy/blob/main/docs/figure_3a.png)
```

Comment:
@tayllatheodoro please replace this schematic with current Fig. 2 in DynaCell draft.